zk_toolbox/crates/zk_inception/src/commands/prover/args/run.rs

```diff
@@ -0,0 +1,87 @@
+use clap::{Parser, ValueEnum};
+use common::PromptSelect;
+use strum::{EnumIter, IntoEnumIterator};
+
+use crate::messages::{MSG_ROUND_SELECT_PROMPT, MSG_RUN_COMPONENT_PROMPT};
+
+#[derive(Debug, Clone, Parser, Default)]
+pub struct ProverRunArgs {
+    #[clap(long)]
+    pub component: Option<ProverComponent>,
+    #[clap(flatten)]
+    pub witness_generator_args: WitnessGeneratorArgs,
```

Comment:
could we move it to the witness generator? 
I assume we can't run different rounds for gateway